django/db/models/sql/query.py

```diff
@@ -1280,6 +1282,31 @@ def _add_q(self, q_object, used_aliases, branch_negated=False,
         needed_inner = joinpromoter.update_join_types(self)
         return target_clause, needed_inner
 
+    def _prepare_lookup(self, lookup):
+        """
+        If the node is carrying generic foreign key lookup and the lookup is
+        simple (no double underscores), convert it to a Q node carrying the
+        double content_type and object_id lookup. Otherwise returns the original
+        lookup
+        :param lookup: a tuple (lookup, value)
+        :return: a transformed lookup - tuple (lookup, value) or a Q object
```

Comment:
Given the logic of this method, I'd reverse the conjuncts here:

```
If the lookup is simple (no double underscores) and the node is carrying generic foreign key lookup...
```

Also, the project doesn't make use of `:param lookup:` and `:return:` annotations.